Index: send.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import socket\r\nimport sys\r\n\r\nHOST = '127.0.0.1'\r\nPORT = 6700\r\nif __name__ == \"__main__\":\r\n    # Create socket for server\r\n    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM, 0)\r\n    # Let's send data through UDP protocol\r\n    send_data = \"Test1\"\r\n    s.sendto(send_data.encode('utf-8'), (HOST, PORT))\r\n    print(\"\\n\\n 1. Client Sent : \", send_data, \"\\n\\n\")\r\n    s.close()\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/send.py b/send.py
--- a/send.py	(revision b352f69b7f5d0c3de4a4f72e35a751218577d15c)
+++ b/send.py	(date 1644388885249)
@@ -8,6 +8,6 @@
     s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM, 0)
     # Let's send data through UDP protocol
     send_data = "Test1"
-    s.sendto(send_data.encode('utf-8'), (HOST, PORT))
+    s.sendto(send_data.encode('utf-8'), ("82.102.57.157", 6700))
     print("\n\n 1. Client Sent : ", send_data, "\n\n")
     s.close()
